src/SonarQube.Client/Api/Requests/V5_10/GetIssuesRequest.cs

```diff
@@ -37,6 +38,13 @@ public class GetIssuesRequest : RequestBase<SonarQubeIssue[]>, IGetIssuesRequest
         [JsonProperty("key")]
         public virtual string ProjectKey { get; set; }
 
+        [JsonProperty("statuses", DefaultValueHandling = DefaultValueHandling.Ignore), DefaultValue(null)]
```

Comment:
This property doesn't exist on the old API so let's make sure it's not serialized nor used.